### ISSUE TITLE
UnusedVariableSniff: False positive with increment operator inside foreach

### ISSUE THREAD

**Issue by carusogabriel** [slevomat/coding-standard#428]
Code to reproduce:
```php
$position = 1;

foreach ($foo as $bar) {
            $quox[] = [
                '@foo' => 'bar',
                'position' => $position++, // Reports unused variable here
                'baz' => [
                    '@id' => $bar,
                ],
            ];
}
```

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/48791724c29374840443dd5404b8dab75cc56a65
